pkg/utils/redis_utils.go

```diff
@@ -14,9 +14,25 @@ import (
 
 const RedisConnTimeoutSecond = 1
 
+func newRedisConn(addr string, connectTimeout time.Duration, readTimeout time.Duration, writeTimeout time.Duration, password string) (redis.Conn, error) {
+	c, err := redis.DialTimeout("tcp", addr, connectTimeout, readTimeout, writeTimeout)
```

Comment:
Yes, Naming NewRedisConn may be better.

Btw, I prefer const RedisConnTimeout = time.Second not 1